lib/roadie/rails/asset_propshaft_provider.rb

```diff
@@ -0,0 +1,35 @@
+module Roadie
+  module Rails
+    class AssetPropshaftProvider
+      include Roadie::AssetProvider
+
+      def initialize
+        @stylesheets = {}
+      end
+
+      def find_stylesheet(stylesheet_asset_path)
+        stylesheet = @stylesheets[stylesheet_asset_path]
+        return stylesheet if stylesheet.present?
+
+        asset = find_asset_in_propshaft(stylesheet_asset_path)
+
+        if asset.present?
+          stylesheet = Stylesheet.new(stylesheet_asset_path, asset.content.force_encoding('UTF-8'))
+          @stylesheets[stylesheet_asset_path] = stylesheet
+        end
+
+        stylesheet
+      end
+
+      def find_asset_in_propshaft(stylesheet_asset_path)
+        digested_name = stylesheet_asset_path.gsub(::Rails.configuration.assets.prefix + '/', '')
+
+        propshaft_stylesheets.detect { |css| css.digested_path.to_s == digested_name }
+      end
+
+      def propshaft_stylesheets
+        ::Rails.application.assets.load_path.assets(content_types: [ Mime::EXTENSION_LOOKUP["css"] ])
+      end
+    end
+  end
+end
```

Comment:
Please add new line at the end of file to prevent unnecessary code diffs in the future
Same for other files thx :)

Reply:
Got it ! Will do it.